steps/sms-notify/sms-confirmation/SmsConfirmation.js

```diff
@@ -11,14 +11,14 @@ class SmsConfirmation extends Question {
         return `sms-notify/sms-confirmation/template`;
     }
 
-    // get i18NextContent() {
-    //     return content;
-    // }
+    get i18NextContent() {
+        return content;
+    }
 
     get form() {}
```

Comment:
No need to override form() if there isn't one - it's safe to remove this (I think).

Reply:
@paulgain I think it needs to be there. It complains when it's being built:
```
NotImplemented: SmsConfirmation must implement form
    at expectImplemented (/usr/src/sya/node_modules/@hmcts/one-per-page/src/errors/expectImplemented.js:25:11)
    at SmsConfirmation.Question (/usr/src/sya/node_modules/@hmcts/one-per-page/src/steps/Question.js:11:5)
    at SmsConfirmation (/usr/src/sya/steps/sms-notify/sms-confirmation/SmsConfirmation.js:4:1)
    at Object.<anonymous> (/usr/src/sya/steps.js:46:5)
    at Module._compile (module.js:569:30)
    at Module._extensions..js (module.js:580:10)
    at Object.nodeDevHook [as .js] (/usr/src/sya/node_modules/node-dev/lib/hook.js:61:7)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/src/sya/app.js:6:15)
    at Module._compile (module.js:569:30)
    at Module._extensions..js (module.js:580:10)
    at Object.nodeDevHook [as .js] (/usr/src/sya/node_modules/node-dev/lib/hook.js:61:7)
[ERROR] 13:27:40 NotImplemented
```

Reply:
Fair enough, I wasn't 100% sure. Looks like it's an issue with the framework, I'll speak to Michael.